activitysim/abm/models/util/test/test_mandatory_tour_frequency.py

```diff
@@ -56,10 +56,11 @@ def test_mtf():
     pdt.assert_series_equal(
         mandatory_tours.person_id,
         pd.Series([10, 20, 20, 30, 30, 40, 40], index=idx, name="person_id"),
+        check_dtype=False,
     )
 
     pdt.assert_series_equal(
-        mandatory_tours.tour_type,
+        mandatory_tours.tour_type.astype(str),
```

Comment:
instead of dumping to `str`, perhaps we should have the test verify the categorical data type here?

Reply:
I see, they are not pd.cat in this unit test any more. This test calls the method in the util/tour_frequency.py. Initially I implemented the pd.cat conversion in the tour_frequency.py, with hardcoded categories. More recently, I moved the pd.cat conversion to the actual tour frequency models themselves, like mandatory_tour_frequency.py, so that the categories can be automatically set based on the UECs instead of hardcode in .py.